docs/manager/graphql-reference/schema.graphql

```diff
@@ -285,6 +288,90 @@ interface Node {
   id: ID!
 }
 
+"""Added in 25.5.0."""
+type AuditLogConnection {
+  """Pagination data for this connection."""
+  pageInfo: PageInfo!
+
+  """Contains the nodes in this connection."""
+  edges: [AuditLogEdge]!
+
+  """Total count of the GQL nodes of the query."""
+  count: Int
+}
+
+"""
+The Relay compliant `PageInfo` type, containing data necessary to paginate this connection.
+"""
+type PageInfo {
+  """When paginating forwards, are there more items?"""
+  hasNextPage: Boolean!
+
+  """When paginating backwards, are there more items?"""
+  hasPreviousPage: Boolean!
+
+  """When paginating backwards, the cursor to continue."""
+  startCursor: String
+
+  """When paginating forwards, the cursor to continue."""
+  endCursor: String
+}
+
+"""Added in 25.5.0. A Relay edge containing a `AuditLog` and its cursor."""
+type AuditLogEdge {
+  """The item at the end of the edge"""
+  node: AuditLogNode
+
+  """A cursor for use in pagination"""
+  cursor: String!
+}
+
+"""Added in 25.5.0."""
+type AuditLogNode implements Node {
+  """The ID of the object"""
+  id: ID!
+  row_id: UUID!
+  entity_type: AuditLogEntityType!
+  operation: ImageAuditLogOperationType!
```

Comment:
Currently, the only Entity type is Image, so the operation type is also limited to Image, which feels a bit awkward. It would be good to find a way to make the operation type more clearly defined.